src/go/rpk/pkg/cli/cmd/cluster/config/export.go

```diff
@@ -53,7 +53,9 @@ func exportConfig(
 		// Preface each property with a descriptive comment
 		var commentTokens []string
 
-		if meta.Example != "" {
+		if len(meta.EnumValues) > 0 {
+			commentTokens = append(commentTokens, fmt.Sprintf("one of %s", strings.Join(meta.EnumValues, ", ")))
```

Comment:
Would this be clearer to be `one of %v", meta.EnumValues`? Or, `one of [%s]`? Both would result in brackets around the strings.

Reply:
The commentTokens are already in a `()` in the string, so I thought it looked a bit cluttered with extra square brackets